app/generated_resources.grd

```diff
@@ -4891,16 +4891,16 @@ Keep your key file in a safe place. You will need it to create new versions of y
         Get back here fast by bookmarking this page
       </message>
       <message name="IDS_INCOGNITOWINDOW_PROMO_0" desc="Variations option 1. Text shown on promotional UI appearing next to the App Menu button, which encourages users to use it.">
-        You can browse privately using an incognito window
+        You can browse privately using a Private Window
```

Comment:
The case in the patch doesn't match the original. Is that right?

Reply:
I'm not even sure that we ever show this Incognito promot bubble that gets anchored to the main menu. But this could kind of UI that we have available could be an interesting idea if we do want to promote knowledge of Private / Tor Windows.